### ISSUE TITLE
Create a guide for PHPStorm

### ISSUE THREAD

**Issue by petrsvihlik** [Kentico/delivery-sdk-php#21]

**Comment by RadoslavCap** [comment]
Dan will review this guide on Ubuntu Linux.

Rocky, please review my English.

**Comment by dusekdan** [comment]
I was able to get this working on Ubuntu Linux with following notes:
- All alternative Ubuntu commands that require `sudo` on Fedora also require `sudo` on Ubuntu
- `apt-get` can be replaced just with `apt`, as it is preferable on newer Ubuntu versions to use this command

I also briefly discussed the language with our TW and he thinks that some KC TW should probably have a look at this tutorial as current 'how to instructions' would be hard to follow for readers.

**Comment by RadoslavCap** [comment]
Fixed Ubuntu commands.

**Comment by petrsvihlik** [comment]
proofreading is done. thx @RadoslavCap !
